clustered/ehcache-common-api/src/main/java/org/ehcache/clustered/common/internal/store/ServerStore.java

```diff
@@ -64,6 +64,7 @@ public interface ServerStore {
    */
   void append(long key, ByteBuffer payLoad) throws TimeoutException;
 
+  void insertChain(long key, Chain chain) throws TimeoutException;
```

Comment:
Javadoc is missing.  (All other methods in this class have javadoc -- this should not be an exception.)